### DISCUSSION TITLE
Install Env Error

### DISCUSSION THREAD

**Discussion by gnaloaixey** [remotebiosensing/rppg#54] [Q&A]
when i run conda env create -f rppg.yaml, it tips me that "Solving environment: failed,ResolvePackageNotFound", how to solve this problem, I am using the window10 environment

**Comment by DrMrli** [comment]
I have the same problem. Have you solved it？

**Comment by NoHairGG** [comment]
My solution:
conda env export --no-builds > environment.yml
conda env create -f environment.yml
(I think) this issue is that the original setup environment is not the same with yours.
References:
https://stackoverflow.com/questions/49154899/resolvepackagenotfound-create-env-using-conda-and-yml-file-on-macos
https://github.com/conda/conda/issues/6073
